README.md

```diff
@@ -30,30 +30,56 @@ On **Bash** and **Zsh** on OS X/Linux:
 ./aws-elastic-beanstalk-cli-setup/scripts/bundled_installer
 ```
 
-**NOTE:** On OS X, if you see installation failing with the following message, "zipimport.ZipImportError: can't decompress data; zlib not available", perform the following exports and then retry the above command:
-
-```bash
-brew install zlib
-export LDFLAGS="-L/usr/local/opt/zlib/lib"
-export CPPFLAGS="-I/usr/local/opt/zlib/include"
-```
 In **PowerShell** or in a **Command Prompt** window:
 
 ```ps1
 .\aws-elastic-beanstalk-cli-setup\scripts\bundled_installer
 ```
 
-**NOTE:** In PowerShell, upon executing `bundled_installer`, if you see an error with the message "execution of scripts is disabled on this system", set the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6) to "RemoteSigned" and then retry the above command.
+#### 2.3. Troubleshooting
 
-```ps1
-Set-ExecutionPolicy RemoteSigned
-```
+- **Linux**
+
+    Almost all installation problems experienced by customers on Linux/OS X so far have been due to missing libraries such as OpenSSL, bzip2, etc., which some OSes (especially minimal Docker containers) don't come with out-of-the-box.
```

Comment:
Remove the "so far" - not necessary and implies that there will be future problems.  We should be confident about what we publish.

Reply:
Okay

Reply:
`OSes` should be `OSs`

Reply:
Okay.